cmd/controller-manager/app/controller-manager.go

```diff
@@ -56,7 +58,7 @@ import (
 )
 
 var (
-	kubeconfig, federationConfig, masterURL string
+	kubeconfig, federationConfig, masterURL, fedNamespace string
```

Comment:
Variable definitions should have the minimal scope required. This variable is package global, but is only used within the scope of a conditional in a function.  As per subsequent comments, though, this variable is not  necessary and can be removed.

Reply:
Yeah, I think that makes sense. I will update this PR accordingly.